pkg/infra/iac2/templates_compiler.go

```diff
@@ -664,6 +664,17 @@ func (tc TemplatesCompiler) handleIaCValue(v core.IaCValue, appliedOutputs *[]Ap
 			tc.getVarName(region), tc.getVarName(v.Resource)), nil
 	case resources.NODE_GROUP_NAME_IAC_VALUE:
 		return fmt.Sprintf(`%s.nodeGroupName`, tc.getVarName(resource)), nil
+	case resources.ALB_CONTROLLER_READINESS_GATE_IAC_VALUE:
+		return `
```

Comment:
would it be helpful to move `testutils.UnIndent` to a prod-scope module, so we could use it here? (Basically just wrap this string with that func, and then you can indent the string and it'll appear un-indented in output.)

Obviously not a blocker.

Reply:
I like that idea, but not as part of this change now that I've removed this `IaCValue`.